joomla/joomla-cms
# [4.3][csp] Reduce inline onchange usage [searchtools]

Pull Request for Issue # .

### Summary of Changes
- remove the filter xml entries `onchange="this.form.submit();"`
- add a class `js-select-submit-on-change` on their place
- Tweet the search tools javascript to replicate the functionality for each element with the class


### Testing Instructions
This PR needs `npm ci`

Pick a list view and check that all the filters are working as before (ie selecting anything with a select element and observe the page refreshing)


### Actual result BEFORE applying this Pull Request

Works

### Expected result AFTER applying this Pull Request

Works

### Link to documentations
Please select:
